### PR TITLE
migrate guide video - registration video

### PR DESCRIPTION
**Pull request by GarnesPratama**

### CODE REVIEW

File: apps/lxp/modules/guide-guide video/module.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import { BreadCrumb } from '@mknows-frontend-services/components/atoms';
+import ContentSection from './section/ContentSection';
+import { ClientProvider } from '../common/provider';
+import { guideVideoBreadCumbs } from './const';
+
+export const RegistrationVideoModule = (): ReactElement => {
+  return (
+    <ClientProvider>
+      <BreadCrumb items={guideVideoBreadCumbs} />
+      <ContentSection />
```
Comment (bangmar) [inline apps/lxp/modules/guide-guide video/module.tsx:12]: footer nya ga pake bang?

**Review by bangmar — CHANGES_REQUESTED** [review]

File: apps/lxp/modules/guide-guide video/section/ContentSection.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import { filterOption } from '../store';
+import { useRouter } from 'next/router';
+
+const ContentSection: FC = (): ReactElement => {
+  const { query } = useRouter();
+  const getOption = useRecoilValue(
+    filterOption(query.videoId as unknown as string)
+  );
+  return (
+    <div>
```
Comment (maulanasdqn) [inline apps/lxp/modules/guide-guide video/section/ContentSection.tsx]: pake Fragment aja

File: apps/lxp/modules/guide-guide video/section/YoutubeSection.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  const onPlayerReady: YouTubeProps['onReady'] = (event: any) => {
+    event.target.pauseVideo();
+  };
+  const opts: YouTubeProps['opts'] = {
+    playerVars: {
+      autoplay: 0,
+    },
+  };
+  return (
+    <div>
```
Comment (maulanasdqn) [inline apps/lxp/modules/guide-guide video/section/YoutubeSection.tsx]: gak usah pake div langsung aja

**Review by maulanasdqn — CHANGES_REQUESTED** [review]
cekcek lagi

**Review by maulanasdqn — APPROVED** [review]

**Review by bangmar — APPROVED** [review]

**Pull request merged**